config.go

```diff
@@ -37,7 +37,10 @@ const (
 	ModeIntegrated = "integrated"
 	ModeRemote     = "remote"
 
-	defaultLndMode = ModeRemote
+	defaultLndMode     = ModeRemote
```

Comment:
pre-existing, but is it correct? Documentation indicates that default is `integrated`

Reply:
It's correct. But we changed the default, so if the doc mentions `integrated` then we need to update it. Where did you find that reference to `integrated`?

Reply:
Ah, I found where it was still wrong. Added a commit to fix it.